### ISSUE TITLE
The time doesn't display in a right way

### ISSUE THREAD

**Issue by ChenQiqian** [theme-next/hexo-symbols-count-time#15]
When the reading time is greater than 1 hours, it will display like this:

![23333](https://raw.githubusercontent.com/ChenQiqian/images/master/t1.png)

I believe that the problem is caused by the mistake in here:

```js 
// /lib/helper.js
var getFormatTime = function(minutes, suffix) {
  var fHours = Math.floor(minutes / 60);
  var fMinutes = Math.floor(minutes - (fHours * 60));
  if (fMinutes < 1) { fMinutes = 1; } // 0 => 1
  if (!suffix) { suffix = 'mins.'; } // 1 => 1 mins.
  return fHours < 1
    ? fMinutes + ' ' + suffix // < 59 => 59 mins.
    : fHours + ':' + (fMinutes < 10 && (fMinutes = '0' + fMinutes)); // = 61 => 1:01
};
``` 

because when I changed this javascript, it worked well.

```js
var getFormatTime = function(minutes, suffix) {
  var fHours = Math.floor(minutes / 60);
  var fMinutes = Math.floor(minutes - (fHours * 60));
  if (fMinutes < 1) { fMinutes = 1; } // 0 => 1
  if (!suffix) { suffix = 'mins.'; } // 1 => 1 mins.
  if (fMinutes < 10) fMinutes = '0' + fMinutes;
  return fHours < 1
    ? fMinutes + ' ' + suffix // < 59 => 59 mins.
    : (fHours + ':' + fMinutes); // = 61 => 1:01
};
```

![???](https://raw.githubusercontent.com/ChenQiqian/images/master/t2.png)

I don't know much about javascript, so I hope that you can fix this bug quickly.

Sorry for my poor English.

**Comment by ChenQiqian** [comment]
Sorry, the right code should be:
```js
var getFormatTime = function(minutes, suffix) {
  var fHours = Math.floor(minutes / 60);
  var fMinutes = Math.floor(minutes - (fHours * 60));
  if (fMinutes < 1) { fMinutes = 1; } // 0 => 1
  if (!suffix) { suffix = 'mins.'; } // 1 => 1 mins.
  if (fHours > 0 && fMinutes < 10) fMinutes = '0' + fMinutes;
  return fHours < 1
    ? fMinutes + ' ' + suffix // < 59 => 59 mins.
    : (fHours + ':' + fMinutes); // = 61 => 1:01
};
```
It may have other bugs, since I haven't learned javascript.

**Comment by ChenQiqian** [comment]
I haven't noticed that pull request fix this bug. 
I thought it only solved the bug when the minutes were less than 10 minutes.
Thanks. @ivan-nginx

**Comment by ivan-nginx** [comment]
@ChenQiqian so, this pull fix your bug or not?

**Comment by ChenQiqian** [comment]
yes.

**Comment by ivan-nginx** [comment]
Ok, thank's!

**Comment by ivan-nginx** [comment]
Fixed in [v0.4.3 release](https://github.com/theme-next/hexo-symbols-count-time/releases/tag/v0.4.3).

If solved, close it please.
